.github/workflows/update-page.yaml

```
@@ -0,0 +1,47 @@
+          else
+            path="${{ inputs.repoName }}"
+            # create dir if it doesn't exist yet
+            [ ! -d "$path" ] && mkdir "$path"
+          fi
+
+          if ! diff -q .nindex/index.yaml "$path/index.yaml"; then
+            cp .nindex/index.yaml "$path/index.yaml"
+            git add "$path/index.yaml"
+            git commit -m "chore(helm-repo): update index.yaml"
+            git push
+          fi 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)